atlas/schema.hcl

```diff
@@ -1,11 +1,399 @@
-table "test" {
+table "expense_snapshot_v1" {
   schema = schema.public
-  column "id" {
+  column "expense_id" {
     null = false
-    type = integer
+    type = text
+  }
+  column "snapshot_id" {
+    null = false
+    type = text
+  }
+  column "scan_id" {
+    null = true
+    type = text
+  }
+  column "source" {
+    null = false
+    type = text
+  }
```

Comment:
i think you can use the comment/description (there was a field) in the column section to document it

Reply:
Like just add a comment elaborating what "source" means?

Reply:
I removed the source column - after thinking about it some more I don't think it's important at this time

- source was just whether or not the expense/income came from a receipt or invoice
- We track the name of the expense/income with title
- we track if it was a result of the inference or manual alteration via if scan_id is null or not

Reply:
can we document this logic in the comments field

Reply:
i think you should document the case if scan id is not null, since there could be many alternatives if it is null, not just manual alterations